src/app/store/entities/sandboxes/boilerplates/default-boilerplates.js

```diff
@@ -1,6 +1,7 @@
 export const JS = {
   id: 'js',
-  condition: '\.js$', // eslint-disable-line
+  extension: '.js',
+  condition: '.js$', // eslint-disable-line
```

Comment:
Ah, this is a leftover from the 286th iteration of CodeSandbox. It's called boilerplates, if you enable module view and view a component that only exports itself it will run the boilerplate for the `.js` file and render the component.

The issue was that I've changed the caching logic to use a special caching strategy on `.js`, but this wasn't propagated to boilerplates.

I'm not planning to add `.jsx` support as `create-react-app` doesn't support this either, I want to keep compatibility with `create-react-app`.